res/shaders/gba-color.shader/gba-color.fs

```diff
@@ -0,0 +1,42 @@
+varying vec2 texCoord;
+uniform sampler2D tex;
+uniform vec2 texSize;
+
+uniform float darken_screen;
+uniform float target_gamma;
+uniform float display_gamma;
+uniform float sat;
+uniform float lum;
+uniform float contrast;
+uniform float blr;
```

Comment:
It could be. I'll play around with your parameter system and see what I can come up with.